app/models/course-stage-solution.ts

```diff
@@ -0,0 +1,22 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+import type CourseStageModel from 'codecrafters-frontend/models/course-stage';
+import type LanguageModel from 'codecrafters-frontend/models/language';
+
+export default class CourseStageSolutionModel extends Model {
+  @attr() declare authorDetails: { [key: string]: string }; // free-form JSON
+  @attr() declare changedFiles: { diff: string; filename: string }[]; // free-form JSON
+  @attr('string') declare explanationMarkdown: string;
+  @attr() declare reviewersDetails: Array<{ [key: string]: string }>; // free-form JSON
```

Comment:
@libmartinito let's add the proper types for `authorDetails` and `reviewersDetails`, check core to find out what they are

Reply:
Oh, actually wait - ignore this since we aren't using it right now